src/main/io/osd.c

```diff
@@ -628,6 +657,11 @@ void osdDrawElements(void)
         osdDrawSingleElement(OSD_HOME_DIR);
     }
 #endif // GPS
+#ifdef MAG
+    if (sensors(SENSOR_MAG) || sensors(SENSOR_GPSMAG)) {
```

Comment:
@mikeller what is the best solution? Change the conditionals for (MAG or GPS) or simply remove them and let it active always?